cclib/parser/data.py

```diff
@@ -155,12 +155,14 @@ class ccData:
        "nmrcouplingtensors":       Attribute(dict,             'NMR spin-spin coupling tensors', 'properties:nmr'),
        "nocoeffs":         Attribute(numpy.ndarray,    'TBD',                         'N/A'),
        "nooccnos":         Attribute(numpy.ndarray,    'TBD',                         'N/A'),
-       "nsocoeffs":         Attribute(list,    'TBD',                         'N/A'),
-       "nsooccnos":         Attribute(list,    'TBD',                         'N/A'),
+       "nsocoeffs":        Attribute(list,             'TBD',                         'N/A'),
+       "nsooccnos":        Attribute(list,             'TBD',                         'N/A'),
        "optdone":          Attribute(list,             'done',                        'optimization'),
        "optstatus":        Attribute(numpy.ndarray,    'status',                      'optimization'),
        "polarizabilities": Attribute(list,             'polarizabilities',            'N/A'),
        "pressure":         Attribute(float,            'pressure',                    'properties'),
+       "processors":       Attribute(int,              'processors',                  'metadata'),
+       "qm_program":       Attribute(str,              'QM program',                  'metadata'),

```

Comment:
I think we want to get rid of these now since
1. `metadata['num_cpus']` from #1171, #1271
2. `qm_program` can be parsed from `str(self)` or something similar (the class name).

Reply:
sorry yes, left these in by mistake.